docker-compose.yml

```diff
@@ -1,60 +1,52 @@
-version: '3'
+version: "3.7"
+
+x-web: &web
+  image: trojstenweb:latest
+  build:
+    context: .
+    dockerfile: ./docker/dev/Dockerfile
+  volumes:
+    - .:/web
+
+x-webenv: &webenv
+  TROJSTENWEB_DATABASE_URL: db
+  TROJSTENWEB_DATABASE_USER: trojsten
+  TROJSTENWEB_DATABASE_NAME: trojsten
+
 services:
   db:
-    image: postgres:10.1-alpine
+    image: postgres:12-alpine
     volumes:
       - db_data:/var/lib/postgresql/data
-    networks:
-      trojsten:
-        ipv4_address: 10.47.0.2
     environment:
       - POSTGRES_USER=trojsten
       - POSTGRES_DB=trojsten
   web:
-    image: trojstenweb:latest
+    <<: *web
     ports:
       - "8000:8000"
-    volumes:
-      - .:/web
-    networks:
-      trojsten:
-        ipv4_address: 10.47.0.3
+    environment:
+      <<: *webenv
+      DJANGO_SETTINGS_MODULE: trojsten.settings.development
+      TROJSTENWEB_LOGIN_PROVIDER_URL: http://10.47.0.4:8047
```

Comment:
we deleted the static IP addressing from the container configuration, but we are still using it here.. we should use the new `TROJSTENWEB_LOGIN_PROVIDER_INTERNAL_URL` here.., right now this doesn't work because the browser tries to open the `10.47.0.4:8047` when clicking on login, but it doesn't point anywhere, because we removed the IP mappings..